resources/views/about.blade.php

```diff
@@ -0,0 +1,12 @@
+@extends('layouts.app')
+
+@section('main-content')
+    <div class="container">
+        <div class="row">
+            <div class="col-6 text-center">
+                <h1>About</h1>
```

Comment:
Ok but write About us instead of just `About`

Reply:
Ok, working on it!